src/controllers/health.check.controller.ts

```diff
@@ -0,0 +1,12 @@
+import { Response, Router, Request } from "express";
+import { handleErrors } from "../middleware/error.handler";
+import { logger } from "../utils/logger";
+
+export const healthCheckController = Router({ mergeParams: true });
+
+async function healthCheck(req: Request, res: Response) {
+    logger.debug(`health check triggered`);
+    await Promise.resolve(res.sendStatus(200));
+}
+
+healthCheckController.get('/', handleErrors(healthCheck));
```

Comment:
In this case `handleErrors` is redundant as the code just returns status 200 with on body or logic and cannot throw an exception. This would also make the async redundant too as there is no async code in the function. 
So code can be simplified to:
```suggestion
function healthCheck() { // I don't think the paramaters are needed when they are not used but you should double check this and re-add them if necessary
    logger.debug(`health check triggered`);
    return res.sendStatus(200);
}

healthCheckController.get('/', healthCheck);
```

Reply:
Thank you - changes are on there way